kuma/users/templates/users/profile.html

```diff
@@ -94,6 +96,13 @@ <h1 class="page-title">
           {% endif %}
       </div>
 
+      {% if profile.is_banned and request.user.is_superuser %}
+      <div class="banned">
+          User banned {{ datetimeformat(profile.active_ban().date, format='date') }} 
+          by {{ profile.active_ban().by }}.
+      </div> 
+      {% endif %}
```

Comment:
Instead of just hiding the ban button and showing the ban info somewhere else, maybe we should just replace the "Ban" button with this "Already banned" info?